modules/ROOT/pages/performance-troubleshooting.adoc

```diff
@@ -1,7 +1,7 @@
 = Performance troubleshooting
 :description: Learn how to monitoring your Bonita Platform and troubleshoot performance issues.
```

Comment:
Learn how to monitoring your Bonita Platform  => Learn how to monitor your Bonita Platform

Reply:
Done